infrastructure/smart-contracts/contracts/upgrades/referee/Referee9.sol

```diff
@@ -534,6 +538,8 @@ contract Referee9 is Initializable, AccessControlEnumerableUpgradeable {
         bytes memory _confirmData
     ) public {
 
+        require(_nodeLicenseId <= MAX_KEY_SOLO_SUBMISSION, "51");
+
```

Comment:
Should we start using the custom error/revert pattern for contract size optimizations?